src/action.c

```diff
@@ -103,7 +103,7 @@ action_create(const char *action_name)
 		wlr_log(WLR_ERROR, "action name not specified");
 		return NULL;
 	}
-	struct action *action = calloc(1, sizeof(struct action));
+	MAKE_NEW(struct action, action);
```

Comment:
I am not a fan if the look of this. I very much prefer `struct action *action = whatever()`